rllib/env/single_agent_env_runner.py

```diff
@@ -63,7 +63,8 @@ def __init__(self, config: AlgorithmConfig, **kwargs):
         """
         super().__init__(config=config)
 
-        self.worker_index = kwargs.get("worker_index")
+        self.worker_index: int = kwargs.get("worker_index")
+        self.tune_trial_id: str = kwargs.get("tune_trial_id")
```

Comment:
For what do we need the trial ID in the env runners? For callbacks?

Reply:
Some users might need this in their env.